fixtures/soucet_kasiopea/judge_bad_whitespace.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+import os
+
+
+def main():
+    test_input_file = os.getenv("TEST_INPUT")
+    test_output_file = os.getenv("TEST_OUTPUT")
+
+    if test_input_file is None or test_output_file is None:
+        exit(2)
+
+    with open(test_input_file, "r") as fin:
+        with open(test_output_file, "r") as fcorrect:
+            t = int(fin.readline())
+
+            for _ in range(t):
+                a, b = [int(x) for x in fin.readline().split()]
+                c = int(fcorrect.readline())
+
+                contestant = input()
+
+                if contestant.endswith("\r"):
+                    # Looks like we didn't handle Windows line-endings correctly!
+                    contestant += "oops!"
+                contestant = int(contestant)
```

Comment:
Nefailne tohle nějak exceptionem? Je to to, co tu chceme?

```python
int("3\roops!") # ~> ValueError
int("3oops!") # ~> ValueError
```

Reply:
Jojo, failne - myslím ale, že na způsobu selhání celkem nezáleží, pokud je return code nenulový. `python3 -c "int('a')"` mi dává return code 1, kterým judge má signalizovat "špatná odpověď". Takže nahradit to `exit(1)` by bylo asi ve výsledku stejné.